examples/cim/rl/env_sampler.py

```diff
@@ -84,10 +84,15 @@ def _post_eval_step(self, cache_element: CacheElement, reward: Dict[Any, float])
 agent2policy = {agent: f"{algorithm}_{agent}.policy" for agent in Env(**env_conf).agent_idx_list}
```

Comment:
If we need to pass `agent2policy` as a parameter when calling `env_sampler_creator`, I suggest we move `agent2policy` to another proper place.

Reply:
Restored to the original signature